Cargo.toml

```diff
@@ -8,3 +8,6 @@ members = [
 lto = true
 debug = 1
 incremental = false
+
+[patch.crates-io]
+glutin = { git = "https://github.com/samdoiron/glutin" }
```

Comment:
Obviously temporary. Will switch to the released version once/if [this PR is merged](https://github.com/rust-windowing/glutin/pull/1341)